addons/easyshare/document/easysharefolder/nuxeo-easysharefolder-create-layout.html

```diff
@@ -61,6 +61,23 @@
       value="{{document.properties.eshare:contactEmail}}"
     >
     </nuxeo-input>
+
+    <paper-checkbox noink name="accessCodeRequired"
+      checked="{{document.properties.eshare:accessCodeRequired}}"
+      id="accessCodeRequired" on-tap="checkAccessCodeRequired">
+      [[i18n('easysharefolder.accessCodeRequired')]]
+    </paper-checkbox>
+
+    <nuxeo-input
+      role="widget"
+      label="[[i18n('easysharefolder.accessCode')]]"
+      id="accessCode"
+      name="accessCode"
+      value="{{document.properties.eshare:accessCode}}"
+      on-value-changed="checkAccessCode"
```

Comment:
We should be able to achieve similar behavior without needing `checkAccessCodeRequired` and `checkAccessCode`, by changing:
```suggestion
      required="[[document.properties.eshare:accessCodeRequired]]"
```

Reply:
OK, tested as expected but regarding checkAccessCodeRequired method launched on-tap, the accessCode input is also changing untapping the checkbox (passing from red to blue and removing the required red star). Do I also need to remove it ?